src/unix/proctitle.c

```diff
@@ -83,15 +81,11 @@ char** uv_setup_args(int argc, char** argv) {
 
 
 int uv_set_process_title(const char* title) {
-  char* new_title;
-  /* Copy the title into our own buffer. We don't want to free the pointer
-   * on libuv shutdown because the program might still be using it. */
-  new_title = uv__strdup(title);
-  if (new_title == NULL)
-    return -ENOMEM;
-  uv__free(process_title.str);
-  process_title.str = new_title;
-  process_title.len = strlen(new_title);
+  if (process_title.len == 0)
+    return 0;
```

Comment:
even if it can't change `argv[0]`, it seems like the `uv__set_process_title` should still be able to succeed?